osf/models/node.py

```diff
@@ -185,7 +185,7 @@ def can_view(self, user=None, private_link=None):
 
 class AbstractNode(DirtyFieldsMixin, TypedModel, AddonModelMixin, IdentifierMixin,
                    NodeLinkMixin, CommentableMixin, SpamOverrideMixin, TaxonomizableMixin,
-                   ContributorMixin, Taggable, Loggable, GuidMixin, BaseModel):
+                   ContributorMixin, Taggable, FileTargetMixin, GuidMixin, BaseModel):
```

Comment:
If you call AbstractNode.objects.all(), you get  `ValueError: Invalid AbstractNode identifier: u'osf.quickfilesnode'`

Reply:
Weirdly this was fixed by importing `QuickfilesNode` to `osf/models/__init__.py` with the other osf app models. Not sure why but it needs it there to detect  `osf.quickfilesnode` as a type.